README.md

```diff
@@ -845,6 +845,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
     * [peewee](https://github.com/coleifer/peewee) - A small, expressive ORM.
     * [pony](https://github.com/ponyorm/pony/) - ORM that provides a generator-oriented interface to SQL.
     * [pydal](https://github.com/web2py/pydal/) - A pure Python Database Abstraction Layer.
+    * [pypika](https://pypi.python.org/pypi/PyPika) - SQL query builder with a pythonic syntax that doesn't limit the expressiveness of SQL.
```

Comment:
It's probably worth using a link to the repo on GitHub rather than link to the package on PyPI.

```suggestion
    * [pypika](https://github.com/kayak/pypika) - SQL query builder with a pythonic syntax that doesn't limit the expressiveness of SQL.
```